.sbtrc

```diff
@@ -0,0 +1,5 @@
+# Test run control
+alias test=; testOnly -l IntegrationTest
+alias testDB=; testOnly -t IntegrationTest
+alias testIT=; testOnly -- -t IntegrationTest
+alias testAll=; testOnly
```

Comment:
```suggestion
alias testAll=; testOnly

```

Reply:
Added missing last empty line in commit 297057f353d2d1c7690637cb175508d48b91ac44.